src/Microsoft.ServiceHub.Analyzers/Types.cs

```diff
@@ -1,30 +1,56 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.ServiceHub.Analyzers
+namespace Microsoft.ServiceHub.Analyzers;
+
+public static class Types
 {
-	internal static class Types
+	public static class IServiceBroker
 	{
-		internal static class IServiceBroker
-		{
-			internal const string FullName = "Microsoft.ServiceHub.Framework.IServiceBroker";
+		public const string FullName = "Microsoft.ServiceHub.Framework.IServiceBroker";
```

Comment:
Do we need a test to ensure these stay correct? Although I imagine if any of these changes, world would break apart.